src/components/SelectionList/Search/ReportListItemHeader.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import Checkbox from '@components/Checkbox';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import {useSearchContext} from '@components/Search/SearchContext';
+import type {ListItem, ReportListItemType} from '@components/SelectionList/types';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {convertToDisplayString} from '@libs/CurrencyUtils';
+import {handleActionButtonPress} from '@userActions/Search';
+import CONST from '@src/CONST';
+import type * as OnyxTypes from '@src/types/onyx';
+import ActionCell from './ActionCell';
+
+type MoneyReportHeaderProps<TItem extends ListItem> = {
+    /** The report currently being looked at */
+    report: OnyxEntry<OnyxTypes.Report>;
+
+    /** The policy tied to the expense report */
+    policy: OnyxEntry<OnyxTypes.Policy>;
+
+    /** Method to trigger when pressing close button of the header */
+    onBackButtonPress: () => void;
+
+    /** The section list item */
+    item: TItem;
+
+    /** Callback to fire when the item is pressed */
+    onSelectRow: (item: TItem) => void;
+
+    /** Callback to fire when a checkbox is pressed */
+    onCheckboxPress?: (item: TItem) => void;
+
+    /** Whether this section items disabled for selection */
+    isDisabled?: boolean | null;
+};
+
+type CellProps = {
+    showTooltip: boolean;
+
+    isLargeScreenWidth: boolean;
+};
+
+type ReportCellProps = {
+    reportItem: ReportListItemType;
+} & CellProps;
```

Comment:
I don't think you need  both `CellProps` and `ReportCellProps` - this is a leftover, can be simplified